server/routes/koala.router.js

```diff
@@ -23,6 +23,20 @@ pool.on('error', (err, client) => {
 });
 
 // GET
+koalaRouter.get('/', (req, res) => {
+
+    let qText = 'Select * FROM "koalalist";';
```

Comment:
your query for creating the table has a capital L: koalaList

Reply:
the table in SQL doesn't respond to the capitalization, I had to debug this